ProjectGroupA/src/main/java/com/fundation/search/model/CustomFile.java

```diff
@@ -21,14 +21,19 @@
 	protected String fileName;
 	protected String ext;
 	protected String size;
-	protected String sizeScale;
+	protected boolean isDirectory;
+	protected boolean isHidden;
+	protected boolean isReadOnly;
 	
 	protected CustomFile() {
 		path = "";
 		fileName = "";
 		ext = "";
 		size = "";
-		sizeScale = "";
+		isDirectory = false;
+		isHidden = false;
+		isReadOnly = false;
+		
 	}
 	
 	protected CustomFile(String path, String fileName, String ext) {
```

Comment:
if this will not be used, please remove.

Reply:
As reviewed with Jose this will not be displayed in the table, thus removing from this class and moving to SearcherCriteria, also verified with Paolo who said "por abstraccion se debe utilizar los atributos necesarios y relevantes para cada clase."